server/src/domain/storage-template/storage-template.service.ts

```diff
@@ -96,7 +96,10 @@ export class StorageTemplateService {
           asset.originalPath = destination;
           asset.sidecarPath = sidecarDestination || null;
         } catch (error: any) {
-          this.logger.warn('Unable to save new originalPath to database, undoing move', error?.stack);
+          this.logger.warn(
```

Comment:
We already log useful info on line 114. Shouldn't we just remove one of these try blocks?

Reply:
This PR isn't solving the issue, just adding better logging. That line won't get called unless we re-throw an error after catching it here. I think we still want to keep to "undo" move operation here, so we can't remove this catch block.